### ISSUE TITLE
Partial profile

### ISSUE THREAD

**Issue by thomas-fossati** [thomas-fossati/draft-psa-token#91]
[Laurence review (part 3)](https://mailarchive.ietf.org/arch/msg/rats/bD5G1ki9f4M31MIpLm0p-odCesc/)

_(I get beat up when I make comments like this, but here goes…) This is a partial profile. It doesn’t guarantee interoperability because it doesn’t lock down algorithms, key identification or freshness. The EAT Constrained Device Standard Profile is not a partial profile like this because it does lock these down. If you leave it as a partial profile, I think you have to call this out. You have to say “further profiling is needed for real client-server interoperation”. Also, if you leave it as is, I think the EAT profile identifier should be removed because it doesn’t identify anything useful._

_My suggestion to deal with the partial profile issue is to define one or more full profiles. Lock down the algorithms and key identification and such for one or more profiles. This doesn’t preclude a future PQ token and such. A PQ token is just a different profile to be defined in the future. It can be easily described as the PSA Token profile with PQ algs._

_Basically, I’d like to discourage partial profiles that don’t interoperate. Maybe the EAT draft should be more clear on this. Maybe it should say that profiles with profile identifiers MUST NOT have any variability that would allow for non-interoperable implementations._

**Comment by thomas-fossati** [comment]
This has now been split into smaller tasks:

* #89 
* #96
* #97
* #95 
* #94 
* #93
* #92
